src/index.html

```diff
@@ -5,8 +5,425 @@
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__review-name">Reviews: 5</div>
+          </div>
+
+          <div class="card__price">
+            <div class="card__price-name">
+              Price:
+            </div>
+            <div class="card__price-number">
+              $2,199
+            </div>
+          </div>
+
+          <a
+            href="#"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            buy
+          </a>
+        </div>
+
+        <div class="card" data-qa="card">
+
+          <img
+            class="card__img"
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)">
+
+          <h2 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+
+          <div class="card__review">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__review-name">Reviews: 5</div>
+          </div>
+
+          <div class="card__price">
+            <div class="card__price-name">
+              Price:
+            </div>
+            <div class="card__price-number">
+              $2,199
+            </div>
+          </div>
+
+          <a
+            href="#"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            buy
+          </a>
+        </div>
+
+        <div class="card" data-qa="card">
+
+          <img
+            class="card__img"
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)">
+
+          <h2 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+
+          <div class="card__review">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="card__review-name">Reviews: 5</div>
+          </div>
+
+          <div class="card__price">
+            <div class="card__price-name">
+              Price:
+            </div>
+            <div class="card__price-number">
+              $2,199
+            </div>
+          </div>
```

Comment:
use semantic tags for text content
```suggestion
          <div class="card__price">
            <p class="card__price-name">
              Price:
            </p>
            <p class="card__price-number">
              $2,199
            </p>
          </div>
```